Tasks/Maven/task.json

```diff
@@ -274,8 +274,17 @@
             "label": "Db User Password",
             "required": false,
             "helpMarkDown": "SonarQube server 5.1 and lower only. The password for the database user i.e. sonar.jdbc.password",
-            "groupName": "SQAnalysis",
+            "groupName": "CodeAnalysis",
             "visibleRule": "sqDbDetailsRequired = true"
+        },
+        {
+          "name": "pmdAnalysisEnabled",
+          "type": "boolean",
+          "label": "Run PMD Analysis",
+          "required": false,
+          "defaultValue": "false",
+          "groupName": "CodeAnalysis",
+          "helpMarkDown": "Use the PMD Java static analysis tool to look for bugs in the code. 'install' or 'package' goals should be executed first."
```

Comment:
Resolved, no action needed.

Running maven with only "jxr:jxr pmd:pmd" as arguments succeeds, so this message can be removed.